distribution/test/itests/test-itests-alliance/src/test/java/org/codice/alliance/test/itests/TestFederation.java

```diff
@@ -197,28 +272,40 @@ public void afterAllianceTest() throws Exception {
             mockServerThread.interrupt();
         }
         mockServerThread = null;
+
+        if (cswServer != null) {
+            cswServer.stop();
+        }
     }
 
     private void startMockResources() throws Exception {
+        cswServer = new FederatedMgmpMockServer(CSW_STUB_SOURCE_ID, INSECURE_ROOT, Integer.parseInt(
```

Comment:
I believe it’s handled by the StubServer.  We use it the same way in DDF : https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java#L251